great_expectations/cli/datasource.py

```diff
@@ -181,6 +181,13 @@ def _build_datasource_intro_string(datasource_count):
     help="Profile ALL data assets within the target data source. "
     "If True, this will override --max_data_assets.",
 )
+@click.option(
+    "--skip_prompt",
+    "-y",
```

Comment:
```suggestion
    "--assume-yes/--yes",
    "-y",
```

Small requested word change here, which will have a cascading (small) change throughout the PR

Reply:
@jcampbell it's fixed!